amplipi/updater/static/index.html

```diff
@@ -18,13 +18,19 @@
     <!-- Custom styles -->
     <link href="static/css/jquery.dm-uploader.min.css" rel="stylesheet">
     <link href="static/css/styles.css" rel="stylesheet">
+
+    <script>
+      function redirectToSettings() {
+        document.getElementById('back-to-app').href = `http://${window.location.hostname}:80/#/settings`;
+      }
+    </script>
   </head>
 
   <body>
 
     <main role="main" class="container">
       <p class="">
-        <a id="back-to-app" class="btn btn-primary btn-sm text-center" role="button" href="/" onclick="javascript:event.target.port=80"><i class="fa fa-arrow-left"></i> Back to App</a>
+        <a id="back-to-app" class="btn btn-primary btn-sm text-center" role="button" href="/" onclick="redirectToSettings()"><i class="fa fa-arrow-left"></i> Back to App</a>
```

Comment:
This change causes tests to fail and stops config upload from working for some reason
Looking into this now

Reply:
It specifically breaks `test_zeroconf` with an assertion error as the last line in the file asserting that `5000 == 9898` and not liking that